src/collective/cover/subscribers.zcml

```diff
@@ -7,13 +7,13 @@
   <!-- event handlers for updating link integrity references -->
 
   <subscriber
-      for="collective.cover.tiles.base.IPersistentCoverTile
+      for="collective.cover.interfaces.ICover
```

Comment:
@rodfersou we need to check this carefully; we should not take care of changes of a working coy until we do a checkin; can you review if this is the event associated with a checkin?